stdlib/timezone/timezone.flux

```diff
@@ -0,0 +1,7 @@
+package timezone
+
+
+utc = {zone: "UTC", offset: 0s}
+fixed = (offset) => ({zone: utc.zone, offset: offset})
```

Comment:
Can we should we consider disallowing mixed duration offsets? If we don't want to enforce it in implementation at least a doc comment explaining that offsets should not be mixed.

Reply:
Mixed duration offsets are generally ok from what I understand. We might want to clarify the order of how they get applied though. But, I'm not entirely sure I understand how the application order effects the end result. The location offset gets applied first in the current implementation.